x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/900_rrf_retriever_nested_query.yml

```diff
@@ -0,0 +1,156 @@
+setup:
+  - requires:
+      cluster_features: 'rrf_retriever_composition_supported'
+      reason: 'test requires rrf retriever composition support'
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
```

Comment:
Maybe leaving a comment on every time you set replica to `0` is just too many comments. Let's see how many more there are...